test/lib/bamboo/adapters/mailgun_helper_test.exs

```diff
@@ -0,0 +1,42 @@
+defmodule Bamboo.MailgunHelperTest do
+  use ExUnit.Case
+  import Bamboo.Email
+  alias Bamboo.MailgunHelper
+
+  # These tests are based mostly on MandrillHelperTest, but seem fragile.
+  # They rely on internal structure used by the adapter and helper, and
+  # don't (AND CAN'T) look at the resulting email created by MailgunAdapter.to_mailgun_body()
+  # without major refactoring.
+  # This is better than nothing, though.
```

Comment:
Similarly, this a great comment to add directly to the PR for the reviewers, or potentially filed as a new issue, but our preference is for it not be a code comment.

Reply:
Comment removed